charts/logfire/values.yaml

```diff
@@ -20,6 +20,8 @@ postgresSecret:
   enabled: false
   # -- Secret name
   name: ""
+  # -- Optional annotations for externally-managed secrets reloading
+  annotations: {}
```

Comment:
perhaps we could make this one and `existingSecret.annotations` a workload specific annotation, like:

```yaml
logfire-backend:
  annotations: {}
```
etc. for every workload.

Even though I like the idea of a convenience wrapper, custom annotations could be added also for other purposes.